lib/runner/test-runners/cucumber.js

```diff
@@ -101,6 +101,26 @@ class CucumberSuite extends TestSuite {
       });
     }
 
+    let requiredModules =  this.argv['require-module'];
+    if (requiredModules) {
+      if (isString(requiredModules)) {
+        requiredModules = [requiredModules];
+      }
+      requiredModules.forEach((module)=>{
+        initialRequires.push('--require-module', module);
+      });
+    }
```

Comment:
this code is being repeated, can we make a common helper function.

Reply:
yes will add that